sources/engine/Stride.Navigation/Processors/RecastNavigationMesh.cs

```diff
@@ -3,62 +3,45 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.Runtime.CompilerServices;
-using Stride.Core;
+using DotRecast.Detour;
 using Stride.Core.Mathematics;
 
 namespace Stride.Navigation.Processors
 {
     /// <summary>
     /// Recast native navigation mesh wrapper
     /// </summary>
-    public unsafe class RecastNavigationMesh : IDisposable
+    public class RecastNavigationMesh(NavigationMesh navigationMesh)
```

Comment:
Is it no longer required to dispose this type, ownership of the NavigationMesh is handler someplace else or no longer requires disposal ?